README.md

```diff
@@ -1,57 +1,113 @@
-# Project Name
-
-(short, 1-3 sentenced, description of the project)
-
-## Features
-
-This project framework provides the following features:
-
-* Feature 1
-* Feature 2
-* ...
-
-## Getting Started
-
-### Prerequisites
-
-(ideally very short, if any)
-
-- OS
-- Library version
-- ...
-
-### Installation
-
-(ideally very short)
-
-- npm install [package name]
-- mvn install
-- ...
-
-### Quickstart
-(Add steps to get up and running quickly)
-
-1. git clone [repository clone url]
-2. cd [respository name]
-3. ...
-
-
-## Demo
-
-A demo app is included to show how to use the project.
-
-To run the demo, follow these steps:
-
-(Add steps to start up the demo)
-
-1.
-2.
-3.
-
-## Resources
-
-(Any additional resources or related projects)
-
-- Link to supporting information
-- Link to similar sample
-- ...
+---
+page_type: sample
+languages:
+- Java
+products:
+- azure
+- azure-netapp-files
+description: "This project demonstrates how to use Java with NetApp Files SDK for Microsoft.NetApp resource provider to deploy a cross-region replication for NFS 4.1 Volume."
+---
+
+# Azure NetAppFiles Cross-Region Replication (CRR) SDK NFS 4.1 Sample Java
+
+This project demonstrates how to deploy a cross-region replication with enabled with NFS 4.1 protocol volume using Java and Azure NetApp Files SDK.
+
+In this sample application we perform the following operations:
+
+* Creation
+  * Primary ANF Account
+	| Primary Capacity pool 
+		| Primary NFS v4.1 Volume 
+		
+ * Secondary ANF Account
+	| Secondary Capacity pool
+		| Secondary NFS v.1 Data Replication Volume with referencing to the primary volume Resource ID
+			
+ * Authorize Source volume with Desitnation Volume Resource ID
+ 
+ * Finally, the clean up process takes place (not enabled by default, please change the variable shouldCleanUp to true at main file if you want the clean up code to take a place),deleting all resources in the reverse order following the hierarchy otherwise we can't remove resources that have nested resources still live. You will also notice that the clean up process uses a function called WaitForNoANFResource, at this moment this is required so we can workaround a current ARM behavior of reporting that the object was deleted when in fact its deletion is still in progress.
+
+
+If you don't already have a Microsoft Azure subscription, you can get a FREE trial account [here](http://go.microsoft.com/fwlink/?LinkId=330212).
+
+## Prerequisites
+
+1. Azure Subscription
```

Comment:
I remember that you had to install mvn in your machine to make this run, if that is the case, we need to add it as a pre-req